client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoConnection.java

```diff
@@ -468,14 +469,22 @@ public CallableStatement prepareCall(String sql, int resultSetType, int resultSe
     public Map<String, Class<?>> getTypeMap()
```

Comment:
Can we implement something similar to SqlServer JDBC driver ? which returns an emptyTypeMap for `getTypeMap` but throws an exception for setting the same.